lib/rubocop/cop/rspec/capybara/visibility_matcher.rb

```diff
@@ -30,18 +30,11 @@ class VisibilityMatcher < Base
           MSG_FALSE = 'Use `:all` or `:hidden` instead of `false`.'
           MSG_TRUE = 'Use `:visible` instead of `true`.'
           CAPYBARA_MATCHER_METHODS = %i[
-            have_selector
-            have_css
-            have_xpath
-            have_link
-            have_button
-            have_field
-            have_select
-            have_table
-            have_checked_field
-            have_unchecked_field
-            have_text
-            have_content
+            have_button have_checked_field have_css have_field have_link
```

Comment:
WDYT of such an approach:
```
%w[button
   checked_field
   css
   field 
   link
   select
   selector
   table
   unchecked_field
   xpath].flat_map { |element| ["have_#{element}", "have_no_#{element}"]}
```
That way we'd never forget to add the no_ version

Reply:
I had to make it a symbol, so I modified it a bit, how do you like it?